src/chrome/content/rules/Better-Business-Bureau.xml

```diff
@@ -1,4 +1,8 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://hurdman.app.bbb.org/ => https://hurdman.app.bbb.org/: (6, 'Could not resolve host: hurdman.bbb.org')
+
```

Comment:
I have seen no indication that ^hurman.bbb.org$ has ever existed. hurman.app.bbb.org still exists. hurman.app.../$ redirects to hurman.bbb.org over http and https, so the effect of our rewrite at worst keeps a man in the middle guessing and at best nil.